sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -3014,6 +3014,48 @@ object GpuOverrides extends Logging {
 
         override val supportBufferConversion: Boolean = true
       }),
+    expr[ApproximatePercentile](
+      "Approximate percentile",
+      ExprChecks.aggNotReduction(
```

Comment:
aggNotReduction says that this works as a window aggregation too, but that is not true. We might need to make a new API. I think that @ttnghia had to do the same thing for stddev so you might want to coordinate.

Reply:
Oh yeah, I almost forget about this. I'll add `groupByOnly` check.